test/unit/old-ui/app/util.spec.js

```diff
@@ -388,4 +403,48 @@ describe('normalizing values', function () {
       assert(resultNotRSK)
     })
   })
+
+  describe('#isInfuraProvider', function () {
+    it('checks, that the given provider is Infura provider', function () {
+      var resultKovan = isInfuraProvider('kovan')
```

Comment:
Does it make sense to simplify this and the other tests into `assert(isInfuraProvider('kovan'))`?

Reply:
@patitonar totally makes sense in order to increase readability. Fixed https://github.com/poanetwork/nifty-wallet/pull/308/commits/3ab54b45cf333f65a465b7fec9522ca94fa223ea